ansible/roles/eos/tasks/ceos_ensure_reachable.yml

```diff
@@ -0,0 +1,49 @@
+- block:
+  - name: wait for container's mgmt-ip reachable
+    wait_for:
+      port: 22
+      host: "{{ ansible_host }}"
+      search_regex: OpenSSH
+      delay: 0
+      timeout: 120
+    register: ip_result
```

Comment:
Is name `ceos_reachability` a better name?

Reply:
modified